aws/sra-test/integration-tests/aws-sdk-s3/tests/sra_test.rs

```diff
@@ -261,6 +263,19 @@ async fn sra_manual_test() {
 "#).unwrap(),
             )]);
 
+    struct ListObjectsV2RetryClassifiers;
+
+    // TODO(orchestrator) perform this step in Codegen
```

Comment:
I've been tagging these with the feature flag
```suggestion
    // TODO(enableNewSmithyRuntime): perform this step in Codegen
```

Is this difficult to code generate? Seems like it should be pretty easy to add to `OperationRuntimePluginGenerator`

Reply:
You're right. IDK why I didn't do this initially. I have now added codegen to set this.